apps/src/templates/sectionsRefresh/AdvancedSettingToggles.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import ToggleSwitch from '@cdo/apps/code-studio/components/ToggleSwitch';
+// import {
+//   assignedUnitTextToSpeechEnabled,
+//   assignedUnitLessonExtrasAvailable
+// } from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+
+export default function AdvancedSettingToggles({
+  updateSection,
+  section,
+
+  // these are currently hard coded into the parent component
+  // In the future we will want to make these dynamic based on the course
+  assignedUnitTextToSpeechEnabled,
+  assignedUnitLessonExtrasAvailable
+}) {
+  const handlePairProgrammingToggle = e => {
+    e.preventDefault;
```

Comment:
if we need this, it should have parenthesis, like `e.preventDefault()`. but if it's working without the parens, maybe this line can be taken out entirely?